crates/re_viewer/src/app.rs

```diff
@@ -639,9 +643,36 @@ impl App {
             UICommand::CopyDirectLink => {
                 self.run_copy_direct_link_command(store_context);
             }
+
+            #[cfg(target_arch = "wasm32")]
+            UICommand::RestartWithWebGl => {
+                if Self::set_url_parameter_and_refresh("renderer", "webgl").is_err() {
+                    re_log::error!("Failed to set URL parameter `renderer=webgl` & refresh page.");
+                }
+            }
+
+            #[cfg(target_arch = "wasm32")]
+            UICommand::RestartWithWebGpu => {
+                if Self::set_url_parameter_and_refresh("renderer", "webgpu").is_err() {
+                    re_log::error!("Failed to set URL parameter `renderer=webgpu` & refresh page.");
+                }
+            }
         }
     }
 
+    #[cfg(target_arch = "wasm32")]
+    fn set_url_parameter_and_refresh(key: &str, value: &str) -> Result<(), wasm_bindgen::JsValue> {
```

Comment:
Does this really belong as a static member of `App`? Maybe put it in a `web_tools.rs` or something instead!

Reply:
good idea, I like that a lot better